tests/foreman/ui/test_contentview.py

```diff
@@ -218,3 +224,104 @@ def test_file_cv_display(session, target_sat, module_org, module_product):
         file_values = session.file.read_cv_table(FAKE_FILE_NEW_NAME)
         assert len(file_values) == 1
         assert file_values[0]['Name'] == cv.name
+
+
+@pytest.mark.upgrade
+@pytest.mark.tier2
+def test_positive_delete_cv_promoted_to_multi_env(
+    session,
+    target_sat,
+    module_org,
+):
+    """Delete the published content view version promoted to multiple
+        environments. Delete the entire content view, with a promoted version to
+        multiple environments.
+
+    :id: f16f2db5-7f5b-4ebb-863e-6c18ff745ce4
+
+    :steps:
+        1. Create and sync yum repository on satellite, add to a new content view.
+        2. Publish the content view, 'Version 1.0'
+        3. Promote the 'Version 1.0' to multiple environments, Library -> DEV.
+        4. Delete the promoted 'Version 1.0', verify removed from environments.
+        5. Publish and promote a new 'Version 2.0' to multiple environments.
+        6. Delete the entire content view, verify removed from environments.
+
+    :expectedresults: The deleted CVV and CV do not exist in ContentViews UI,
+        4. Deleting the single promoted CVV, removed the CV from multiple environments.
+        6. Deleting the entire CV containing promoted CVV, removed the CV from multiple environments.
+
+    :CaseImportance: High
+    """
+    repo = target_sat.cli_factory.RepositoryCollection(
+        repositories=[target_sat.cli_factory.YumRepository(url=settings.repos.yum_0.url)]
+    )
+    repo.setup(module_org.id, synchronize=False)  # sync=True later
```

Comment:
Forgotten debug comment probably.
```suggestion
    repo.setup(module_org.id, synchronize=True)
```